docs/build/web.md

```diff
@@ -60,10 +62,22 @@ There are 2 steps to build ONNX Runtime Web:
 
 ### Build Instructions
 
+ONNX Runtime WebAssembly can be built with or without multi-thread and Single Instruction Multiple Data (SIMD) support.
+This support is added/removed by appending the following flags to the build command, the default build option is without.
+
+| build flag              | usage                           |
+| ----------------------- | ------------------------------- |
+| `--enable_wasm_threads` | build with multi-thread support |
+| `--enable_wasm_simd`    | build with SIMD support         |
+
+A complete build for ONNX runtime WebAssembly artifacts will contain 4 ".wasm" files, covering all ON/OFF configuration of these flags.
```

Comment:
A complete build for ONNX runtime WebAssembly artifacts will contain 4 ".wasm" files **with a few ".js" files as listed below:**

/// put the table here